SpatialGDKEditorToolbar/Source/Private/SpatialGDKEditorGenerateSnapshot.cpp

```diff
@@ -457,28 +457,38 @@ bool SpatialGDKGenerateSnapshot(UWorld* World)
 	Worker_SnapshotParameters Parameters{};
 	Parameters.default_component_vtable = &DefaultVtable;
 	Worker_SnapshotOutputStream* OutputStream = Worker_SnapshotOutputStream_Create(TCHAR_TO_UTF8(*SavePath), &Parameters);
+	if (const char* SchemaError = Worker_SnapshotOutputStream_GetError(OutputStream))
```

Comment:
What if we pass the OutputStream into a new function that calls each sub-function and returns the result, and we just destroy the stream once in the calling function?